### DISCUSSION TITLE
Overview of Papers by ANT Workgroup

### DISCUSSION THREAD

**Discussion by edzob** [JRBliekendaal/master-thesis#29] [General]
Perphaps that in the pdf of 2021-2019 a few titles can be found that add value to the research question.
These are resources I did not include in my research.
    
## International Conference on Ambient Systems, Networks and Technologies
List of ANT workgroup programm (author, title) in pdf.

1. http://cs-conferences.acadiau.ca/ant-21/#conferenceProgram
        1. http://cs-conferences.acadiau.ca/ant-21/subPages/ANT_EDI40_Program_2021.pdf
1. http://cs-conferences.acadiau.ca/ant-20/#conferenceProgram
        1. http://cs-conferences.acadiau.ca/ant-20/subPages/ANT_EDI40_Program_2020.pdf
1. http://cs-conferences.acadiau.ca/ant-19/#conferenceProgram
        1. http://cs-conferences.acadiau.ca/ant-19/subPages/ANT_EDI40_Program_2019.pdf
1. http://cs-conferences.acadiau.ca/ant-18/#conferenceProgram
        1. http://cs-conferences.acadiau.ca/ant-18/subPages/ANT_SEIT_Program_2018.pdf
1. http://cs-conferences.acadiau.ca/ant-17/#conferenceProgram
        1. http://cs-conferences.acadiau.ca/ant-17/subPages/ANT_SEIT_Program_2017.pdf
1. http://cs-conferences.acadiau.ca/ant-16/#conferenceProgram
        1. http://cs-conferences.acadiau.ca/ant-16/subPages/ANT_SEIT_Program_2016.pdf
1. http://cs-conferences.acadiau.ca/ant-15/#conferenceProgram
        1. http://cs-conferences.acadiau.ca/ant-15/subPages/ANT_SEIT_Program_2015.pdf
1. http://cs-conferences.acadiau.ca/ant-14/#conferenceProgram
        1. http://cs-conferences.acadiau.ca/ant-14/subPages/2014_Advanced_Program.pdf

**Comment by JRBliekendaal** [comment]
Already had No More Snake Oil from 2019 on the list. Now I also pulled 2 others from O'Reilly to be evaluated.

A new refreshing look on the subject with a new angle of the Residuality Theory.

Also some direct related papers towards EA.. the rest are less usefull.

**Comment by edzob** [comment]
Barry has the same "angre fire" burning as does taleb.
It provides him with great energy and insight :)
I you want an email address, let me know. He is not that fast in responding (very very busy)
He is very talented, smart and experienced. 

Not certain what I think about the direction he is going into.. but knowing him the result will be very interresting.

**Reply by JRBliekendaal** [reply]
I will scan his articles tonight/tomorrow. When I want to have contact with him I will report back ;)

**Reply by JRBliekendaal** [reply]
@edzob Barry has the same mindset as I have.

Resilient systems are, by definition, able to survive disruption and eventually regain function. Beyond resilience is the
idea of antifragility – that systems actually learn from their exposure to stress and become stronger because of it. (O'Reilly, 2020)

With others I read a lot that eventually CAS is the end state for getting into an antifragile mode. While I think that beyond CAS you have Antifragile (same opinion as you have btw)

**Reply by JRBliekendaal** [reply]
Also seeing another nice link to our theories... On page 3 of his introduction:

The Process of Residual Analysis
• Produce a naïve architecture. A functional solution on any platform using current methods from OOP/SOA,
etc.
• Describe the system as a set of flows of information between actors. This breaks the system down into units
of communication that prevent assumptions about use cases or processes and allows us to investigate
contagion under stress.
• List stressors that impact the system
• Describe the residues and how we can add functions to each to allow the residue to survive the stressor that
defines it. We do this without recourse to probability; we cover a large number of possibilities and the residual
(and more...)

Bullet 2 gives me an idea that you can use DEMO to accomplish this with the transactions (Habermas) (like a simple OCD) :) Maybe even it is the case that, because demo is about the construction and not about the function, that a transaction is the residu Barry is talking about. So when you start with EE you already know your residuals.

**Comment by edzob** [comment]
Bullet 2 > "• Describe the system as a set of flows of information between actors. This breaks the system down into units
of communication that prevent assumptions about use cases or processes and allows us to investigate
contagion under stress." 
> Bullet 2 gives me an idea that you can use DEMO to accomplish this with the transactions (Habermas) (like a simple OCD) :) 
Yes and there is also the overlap between DEMO and NST .. Decoupling by normalizing (summarizing) the information flow.

DEMO and NST have the "fragility" of the input (communication) and the "arbitrairy-ness" of the end-result.
for example: give 2 persons the same set of documents and they find different OCD's, 
and add 1 document to the set and the NST Normalisation needs to restart and find different "normalized functions"

and yes. the power of the residu is where it all boils down to.
This then is the lens of IT systems as store of information facts. 
As Martin Op 't Land stresses.

This is the benefit of limiting scope to IT Systems and excluding the function and the organisation from the scope..

**Reply by JRBliekendaal** [reply]
Btw you said that you have Barry his personal email. Can you share it? It could be that I will give him a call somewhere between this week and a month. Really want to know if he is already further with his theory on residuality and I want to talk with him on the applicability of this theory on the enterprise architecture capability / organisation.

**Comment by edzob** [comment]
Just double checked. His email is public
barry@blacktulip.se

https://www.dfkompetens.se/webroot/documents/58/Barry%20O'Reilly.pdf
https://www.cutter.com/article/apply-residuality-theory-improve-systems

and barry on twitter https://twitter.com/technologytulip
